plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseJdbcConnectorTest.java

```diff
@@ -1286,6 +1286,9 @@ public void testDeleteWithVarcharPredicate()
     @Test(dataProvider = "testInsertBatchSizeSessionProperty")
     public void testInsertBatchSizeSessionProperty(Integer batchSize, Integer numberOfRows)
     {
+        if (!hasBehavior(SUPPORTS_CREATE_TABLE)) {
+            return;
```

Comment:
nit: Prefer throwing a `SkipException` instead. That way passing tests can be distinguished from tests which are intentionally skipped.

For example something like `throw new SkipException("Unable to CREATE TABLE to test insert batch size");`?

Reply:
done